Procfile.example

```diff
@@ -45,9 +45,9 @@
 # These workers subscribe to a queue named based on the particular EMS they connect to
 # The "<id>" tag in these lines should be replaced by the id of the manager instance these workers apply to
 
-# vmware_refresh_<id>:      env QUEUE=ems_<id> ruby lib/workers/bin/run_single_worker.rb ManageIQ::Providers::Vmware::InfraManager::RefreshWorker
-# vmware_event_<id>:        env QUEUE=ems_<id> ruby lib/workers/bin/run_single_worker.rb ManageIQ::Providers::Vmware::InfraManager::EventCatcher
-# vmware_refresh_core_<id>: env QUEUE=ems_<id> ruby lib/workers/bin/run_single_worker.rb MiqEmsRefreshCoreWorker
+# vmware_refresh_<id>:      env EMS_ID=<id> ruby lib/workers/bin/run_single_worker.rb ManageIQ::Providers::Vmware::InfraManager::RefreshWorker
+# vmware_event_<id>:        env EMS_ID=<id> ruby lib/workers/bin/run_single_worker.rb ManageIQ::Providers::Vmware::InfraManager::EventCatcher
+# vmware_refresh_core_<id>: env EMS_ID=<id> ruby lib/workers/bin/run_single_worker.rb MiqEmsRefreshCoreWorker
```

Comment:
@carbonin ^ this 'seems' to work locally

Reply:
Locally is the only use case for the Procfile so :+1: to that.